experimental/packages/otlp-exporter-base/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "@opentelemetry/otlp-exporter-base",
+  "version": "0.27.0",
+  "description": "OpenTelemetry SDK for Node.js",
+  "main": "build/src/index.js",
+  "module": "build/esm/index.js",
+  "esnext": "build/esnext/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js",
+  "browser": {
+    "./src/platform/index.ts": "./src/platform/browser/index.ts",
+    "./build/esm/platform/index.js": "./build/esm/platform/browser/index.js",
+    "./build/esnext/platform/index.js": "./build/esnext/platform/browser/index.js",
+    "./build/src/platform/index.js": "./build/src/platform/browser/index.js"
+  },
+  "scripts": {
+    "prepublishOnly": "npm run compile",
+    "compile": "tsc --build tsconfig.all.json",
+    "clean": "tsc --build --clean tsconfig.all.json",
+    "codecov:browser": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
```

Comment:
Is it intended that there is no `codecov` script (only codecov:browser)?
Seems same applies to some other experimental packages.

Reply:
Hmm no. I'll fix that in this package here and make another PR to fix in others.

Reply:
it looks like the codecov isn't working for experimental at all. instead of fixing an experimental specific issue and making CI changes, I'll wait until #2892 is merged and fix it properly.